Datasets/data-connect-dataset-registereddevices.md

```diff
@@ -0,0 +1,120 @@
+- Analyze the registered users devices. 
+- Analyze the management channel of different devices registered by the users. 
+- Analyze the details of onPrem devices registered for every user. 
+
+## Questions
+
+The following are examples of questions that you can answer with this dataset:
+
+- How many total devices are registered by the users of the tenant?
+- Which devices are registered by a particular user in the organization?
+- Are all the device configurations reflected in the status of the device?
+- What are the details of a particular device (like operating system, version) registered by any user?
+- How many registered devices are of a particular configuration/type? 
+
+## Joining with other datasets
+
+The RegisteredDevices_v0 dataset can be joined with relevant users, and other category datasets.
+
+## Definitions
+
+- A user may register some personal devices as well as be registered on some company devices. Devices registered by the user can have certain configurations and specifications that are stored. The configs can be set up in any one of the management channels opted for the particular device. The details of the onPrem devices are also stored and can be retreived by the tenants. 
+
+## Schema
+
+| Name  | Type  |  Description  |  FilterOptions  |  FilterType  |
+| ----------- | ----------- | ----------- | ----------- | ----------- |
+| accountEnabled | boolean | True if the account is enabled, false otherwise. Default is true. | No | None |
+| approximateLastSignInDateTime | datetime | The timestamp of last sign in that represents information in UTC time. | No | None |
+| deviceCategory | string | User defined property set by intune to automatically add devices to groups and simplify managing devices. | No | None |
+| deviceId | string | Unique identifier set by Azure Device Registration Service at the time of registration. This is an alternate key that can be used to refrence the device object. | No | None |
+| deviceOwnership | string | Ownership of the device. This property is set by intune. Possible values are :- unkown, company, personal. | No | None |
+| displayName | string | The display name for the device. | No | None |
+| domainName | string | The on-prem domain name of hybrid azure AD joined devices. This property is set by intune. | No | None |
+| enrollmentProfileName | string | Enrollment profile applied to the device. For example, apple device enrollment profile, Device enrollment - Corporate device identifilers, or Windows autopilot profile name. This property is set by intune. | No | None |
+| enrollmentType | object | Enrollment type of the device. This property is set by intune. Possible values are :- unkown, userEnrollment, deviceEnrollmentManager, appleBulkWithUser, appleBulkWithoutUser, windowsAzureADJoin, windowsBulkuserless, windowsAutoEnrollment, windowsBulkAzureDomainJoin, windowsCoManagement. | No | None |
```

Comment:
I see this is string in json code: https://dev.azure.com/O365Exchange/O365%20Core/_git/MARS?path=/sources/dev/mars/src/DiscoveryService/Data/BasicDataSet_v0_RegisteredDevices_v0.json&version=GBmaster&line=71&lineEnd=71&lineStartColumn=18&lineEndColumn=24&lineStyle=plain&_a=contents

Reply:
Missed changing type while copying table rows. Fixed in latest commit